### PR TITLE
Updated Events page

### LINKED ISSUE #31: Events page responsiblity
The events page i.e. http://lugnitdgp.github.io/events.html is not having correct information on the events. And moreover someone has to come forward to update the events page as soon as its decided. Since, it needs admin rights to add an event kindly request for it if you are interested.

### PR DESCRIPTION
**Pull request by vartika-arora**
Fixes #31

### CODE REVIEW

File: _events/2017-08-14-screencasts.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Screencasts
-date: 2017-08-14 16:59 +0530
-thumbnail: poster.jpg
-description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut venenatis, leo in mollis bibendum, nunc mi molestie nunc, in dapibus mi orci non lorem. Nam erat tellus, sagittis vel urna ac, laoreet efficitur risus. Aliquam sed imperdiet quam, nec auctor neque.
+date: 2017-08-22 00:00 +0530
+thumbnail: screencast_printout.jpg
+description: So you're new to college and ready to embark upon your tech journey but still think that Windows is the best OS and Django is just a movie? Worry not! The GNU/Linux User's Group, NIT Durgapur brings you the Screencast series. Targeted at giving you a brief and precise introduction to the marvel that is Linux and all trending topis in technology, these videos will give your tech knowledge a whole new dimension. From unforking the power of Git and Github to the revolution of Linux, we have covered everything. So open your mind to open source and check out our videos.
```
Comment (Compro-Prasad) [inline _events/2017-08-14-screencasts.md]: Correct the spelling "topis" in "trending topis"

**Comment by Compro-Prasad** [comment]
The `description` of the events should be written within 80 columns which is the standard width of terminals on most platforms. If its generated by netlify then no need to change.

**Comment by ghoshbishakh** [comment]
I don't get this. Was this not supposed to be editable through netlify? @ankan17 @Compro-Prasad

**Comment by ankan17** [comment]
Should we close this?

**Comment by ghoshbishakh** [comment]
yes.. this should be done via netlify afaik.. closing... thanks @vartika-arora

**Pull request closed without merging**